app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag(password_resets_path) do
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = label_tag(:password, "New Password")
+          .row
+            .col-sm-4
+              = text_field_tag(:password, nil, class: "form-control", type: "password")
```

Comment:
This should be definitely `password_field_tag`. You should take a look what `_field_tags` are available in Rails API.